charts/runlocal.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -eu
+chart=./bulk-scan-orchestrator/
```

Comment:
other charts are using a makefile for this, any reason to use a shell script instead?

Reply:
This was to pick up our own orchestrator and deploy it from your machine .. Tbh we will just use draft in the future it is more just a helpful script.
I don't see the purpose to have a makefile under a gradle project, if we really wanted to integrate it we should have a gradle target to deploy our chart from local.